gdal/src/main/scala/geotrellis/contrib/vlm/gdal/GDALDataset.scala

```diff
@@ -68,7 +68,7 @@ case class GDALDataset(token: Long) extends AnyVal {
     val extent = this.extent(dataset)
 
     if (GDALWarp.get_overview_widths_heights(token, dataset, numberOfAttempts, 1, widths, heights) <= 0)
-      throw new Exception("get_overview_widths_heights")
+      throw new MalformedDataException("Unable to construct overviews for resample")
```

Comment:
just a minor thing, mb contruct `overviews RasterExtents` (pr smth in a better language)? Since here we don't construct overviews, just overviews `RasterExtents`.